src/ServiceControl.Audit/Infrastructure/SetupBootstrapper.cs

```diff
@@ -61,52 +56,12 @@ public async Task Run(string username)
                 //No need to start the raw endpoint to create queues
                 await RawEndpoint.Create(config).ConfigureAwait(false);
             }
-
-            var configuration = new EndpointConfiguration(settings.ServiceName);
-            var assemblyScanner = configuration.AssemblyScanner();
-            assemblyScanner.ExcludeAssemblies("ServiceControl.Plugin");
-            if (excludeAssemblies != null)
-            {
-                assemblyScanner.ExcludeAssemblies(excludeAssemblies);
-            }
-
-            configuration.EnableInstallers(username);
```

Comment:
doesn't that mean that we're no longer running the [CreateEventSource](https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl.Audit/Infrastructure/Installers/CreateEventSource.cs) installer?